server/channels/store/sqlstore/store.go

```diff
@@ -1002,18 +1003,57 @@ func (ss *SqlStore) DropAllTables() {
 			    FROM   pg_class
 			    WHERE  relkind = 'r'  -- only tables
 			    AND    relnamespace = 'public'::regnamespace
-				AND NOT relname = 'db_migrations'
+				AND NOT (
+                  relname = 'db_migrations' OR
+                  relname = 'focalboard_schema_migrations' OR
+                  relname = 'focalboard_boards' OR
+                  relname = 'focalboard_blocks'
```

Comment:
Can you elaborate on what's going on here?

Reply:
We avoid removing `focalboard_schema_migrations` for the same reasons we avoid `db_migrations`, not to run migrations for the next test.

As per boards and blocks, during boards product startup we insert the default templates if necessary, which takes about 2 seconds per test, accounting for almost 20min in total of test time for each CI run. This change avoids deleting the blocks and boards related to the templates, so the boards product finds them on start and doesn't run the insert default template process, speeding up the tests.